recipes/numpy_indexed/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set version = "0.3.2" %}
+
+package:
+  name: numpy-indexed
```

Comment:
Please let me know in this PR ( https://github.com/conda-forge/staged-recipes/pull/803 ).